packages/theming/theming-utils/src/mapPipelineToTheme.ts

```diff
@@ -112,6 +112,9 @@ export function mapPipelineToTheme(pipelineOutput: any): AliasColorTokens {
     compoundBrandStroke1Pressed: pipelineOutput.compoundBrandStroke1.strokeColorPressed,
     neutralStrokeDisabled: pipelineOutput.neutralStrokeDisabled.strokeColorRest,
 
+    neutralBackgroundLightStatic: pipelineOutput.neutralBackgroundLightStatic.fillColorRest,
```

Comment:
We should not add values which are android specific on themes here, it can be limited to the android file.

Reply:
Ok make sense , just a thought will iOS will create their specific file ?
[mapPipelineToTheme.ts](https://github.com/microsoft/fluentui-react-native/pull/2372/files/ed1538ec77911880df86b3e95208e9f137b8080c#diff-9b5eeaa8397d44a2c9ceaefd591fe2936905759e53fb06dfa70bc1c014c2348c)

Reply:
Removed this from Common mapPipelineTheme.ts file.